src/app/_components/home-page/LocationMenu.tsx

```diff
@@ -0,0 +1,62 @@
+                            <ul className="px-0">
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's Hikari</a></li>
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's Giga Mall</a></li>
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's Xuan Thuy</a></li>
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's Saigon Pearl</a></li>
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's landmark 72</a></li>
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's Lotter Center</a></li>
+                                <li><a href="" className="text-[1.1713vw]">Pizza 4P's Vincom Pham Ngoc Thach</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div className="border-b-2 border-b-white cursor-pointer" onClick={() => setIsActiveShopList(!isActiveShopList)}>
+                        <span className="text-[1.1713vw] opacity-50 text-white">Select Shop</span>
+                    </div>
+                    <div className="relative w-full h-auto mt-10 after:content-[''] after:h-full after:absolute after:top-0 after:left-0 after:w-full after:bg-[rgb(0,0,0)]/30">
+                        <a href="" className="absolute text-[1.46413vw] text-white top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 z-[1] hover:after:hidden after:content-[''] after:h-[2px] after:absolute after:bottom-0 after:left-0 after:w-full after:bg-white">SHOP LIST</a>
+                        <Image src={'/img/shoplist.jpg'} alt=""
+                            width="0"
+                            height="0"
+                            sizes="100vw"
+                            className="w-full h-auto" />
+                    </div>
+
+                </div>
+            </div>
+            <div className="bg-[#EAE6DC] w-1/2 pt-[56px] pb-[64px] px-[5.85652vw]">
+                <h2 className="text-[4.39239vw] text-white text-center mb-16">Menu</h2>
+                <div className="relative w-full h-0 pb-[100%]">
+                    <div className="overflow-hidden h-0 pb-[100%] z-[1] after:content-[''] after:h-full after:absolute after:top-0 after:left-0 after:w-full after:bg-[rgb(0,0,0)]/20">
+                        <video className="min-h-full min-w-full h-auto w-auto absolute block top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2" src="./video/menu_pizza4ps-1.mp4" loop={true} muted={true} autoPlay={true} playsInline={true}></video>
+
+                    </div>
+                    <div className="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
+                        <ButtonView title="MENU" url="" color="white" />
+                    </div>
+                </div>
+            </div>
+        </section>
+    )
+}
+
+export default LocationMenu
```

Comment:
```suggestion
export default LocationMenu

```
please enter the row